cmd/lncli/commands.go

```diff
@@ -1822,13 +1830,21 @@ func listChainTxns(ctx *cli.Context) error {
 
 	req := &lnrpc.GetTransactionsRequest{}
 
+	start_height := ctx.Int64("start_height")
```

Comment:
this will fail if `start_height` is set and `end_height` is not. It will also fail if end_height is set to `-1`